docs/sources/datasources/prometheus/_index.md

```diff
@@ -143,39 +111,29 @@ To connect the Prometheus data source to Amazon Managed Service for Prometheus u
 
 If you run Grafana in an Amazon EKS cluster, follow the AWS guide to [Query using Grafana running in an Amazon EKS cluster](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-onboard-query-grafana-7.3.html).
 
-### Configure exemplars
+## Exemplars
+
+Exemplars are unique to Prometheus. Exemplars associate higher-cardinality metadata from a specific event with traditional time series data. See [Introduction to exemplars](https://grafana.com/docs/grafana/latest/fundamentals/exemplars/) in Prometheus documentation for detailed information on how they work.
 
 {{% admonition type="note" %}}
 Available in Prometheus v2.26 and higher with Grafana v7.4 and higher.
 {{% /admonition %}}
 
 Grafana 7.4 and higher can show exemplars data alongside a metric both in Explore and in Dashboards.
```

Comment:
We version our docs. We don't need to say this.

Reply:
leaving as is for now - I don't think the versioning here is misleading